pom.xml

```diff
@@ -16,7 +16,7 @@
 
   <properties>
     <jenkins.version>2.13</jenkins.version>
```

Comment:
`jenkins.version` must be at least 2.60.3 then (same in your plugin, missed it in the review)